### ISSUE TITLE
crash on static suite GotG page in DOMStorage

### ISSUE THREAD

**Issue by metajack** [servo/servo#5602]
```[jack@ocate servo]$ ./mach run --release -Z show-parallel-paint ../static-suite/wikipedia/Guardians\ of\ the\ Galaxy\ \(film\)\ -\ Wikipedia\,\ the\ free\ encyclopedia.html 
ERROR:js::rust: Error at file:///home/jack/src/servo/../static-suite/wikipedia/Guardians%20of%20the%20Galaxy%20%28film%29%20-%20Wikipedia,%20the%20free%20encyclopedia_files/load_003.php:89: TypeError: document.write is not a function

JQMIGRATE: Logging is active
ERROR:js::rust: Error at file:///home/jack/src/servo/../static-suite/wikipedia/Guardians%20of%20the%20Galaxy%20%28film%29%20-%20Wikipedia,%20the%20free%20encyclopedia_files/load.php:169: TypeError: document.write is not a function

ERROR:js::rust: Error at file:///home/jack/src/servo/../static-suite/wikipedia/Guardians%20of%20the%20Galaxy%20%28film%29%20-%20Wikipedia,%20the%20free%20encyclopedia_files/load.php:167: ReferenceError: Image is not defined

thread 'StorageManager' panicked at 'called `Option::unwrap()` on a `None` value', /home/larsberg/rust/src/libcore/option.rs:362
stack backtrace:
   1:     0x7f7e00efac92 - sys::backtrace::write::h754dbf5d2d3172ebPBA
   2:     0x7f7e00f0db72 - panicking::on_panic::hdb9e837ac064f8afiHJ
   3:     0x7f7e00edd519 - rt::unwind::begin_unwind_inner::h488386ed3236ccf5lnJ
   4:     0x7f7e00edd8c1 - rt::unwind::begin_unwind_fmt::hac94a0d8ab692f55WlJ
   5:     0x7f7e00f0d497 - rust_begin_unwind
   6:     0x7f7e00f3a204 - panicking::panic_fmt::hf7c6f0a83af17872HZs
   7:     0x7f7e00f3996d - panicking::panic::h279ea5fd08f6f313TXs
   8:     0x7f7e00a08104 - storage_task::StorageManager::start::h2b36ca99e918f472ORf
   9:     0x7f7e00a01694 - thunk::F.Invoke<A, R>::invoke::h433413210190398929
  10:     0x7f7e00988ba0 - rt::unwind::try::try_fn::h15018902619179574208
  11:     0x7f7e00f17a48 - rust_try_inner
  12:     0x7f7e00f17a35 - rust_try
  13:     0x7f7e0098932f - thunk::F.Invoke<A, R>::invoke::h8376645772434066622
  14:     0x7f7e00efdc65 - sys::thread::thread_start::h04d056c6180f4496s8E
  15:     0x7f7dff947529 - start_thread
  16:     0x7f7dfe43722c - clone
  17:                0x0 - <unknown>
thread 'ScriptTask' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', /home/larsberg/rust/src/libcore/result.rs:744
stack backtrace:
   1:     0x7f7e00efac92 - sys::backtrace::write::h754dbf5d2d3172ebPBA
   2:     0x7f7e00f0db72 - panicking::on_panic::hdb9e837ac064f8afiHJ
   3:     0x7f7e00edd519 - rt::unwind::begin_unwind_inner::h488386ed3236ccf5lnJ
   4:     0x7f7e00edd8c1 - rt::unwind::begin_unwind_fmt::hac94a0d8ab692f55WlJ
   5:     0x7f7e00f0d497 - rust_begin_unwind
   6:     0x7f7e00f3a204 - panicking::panic_fmt::hf7c6f0a83af17872HZs
   7:     0x7f7e00465457 - dom::storage::JSRef<'a, Storage>.StorageMethods::RemoveItem::h6462859d2cbdcc24fis
   8:     0x7f7e00464d66 - dom::bindings::codegen::Bindings::StorageBinding::removeItem::hd808407de610edf4Szr
   9:     0x7f7e00f4184a - CallJitMethodOp
  10:     0x7f7e004673db - dom::bindings::codegen::Bindings::StorageBinding::genericMethod::h3efffe130f95bfdfKCr
  11:     0x7f7e006cdd96 - _ZN2js12InvokeKernelEP9JSContextN2JS8CallArgsENS_14MaybeConstructE
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jscntxtinlines.h:389
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsinterp.cpp:352
  12:     0x7f7e006c3b61 - _ZN2js9InterpretEP9JSContextPNS_10StackFrameENS_10InterpModeE
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsinterp.cpp:2414
  13:     0x7f7e006cdca4 - _ZN2js9RunScriptEP9JSContextP8JSScriptPNS_10StackFrameE
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsinterp.cpp:309
  14:     0x7f7e006cdf50 - _ZN2js12InvokeKernelEP9JSContextN2JS8CallArgsENS_14MaybeConstructE
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsinterp.cpp:363
  15:     0x7f7e006ce234 - _ZN2js6InvokeEP9JSContextRKN2JS5ValueES5_jPS3_S6_
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsinterp.h:119
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsinterp.cpp:396
  16:     0x7f7e00731d43 - JS_CallFunctionValue
                        at /home/jack/.cargo/git/checkouts/mozjs-2284529523a8d467/master/js/src/jsapi.cpp:5905
  17:     0x7f7e003fd007 - dom::bindings::codegen::Bindings::FunctionBinding::Function::Call::h4b665161d3950932VbK
  18:     0x7f7e005c6457 - dom::window::JSRef<'a, Window>.WindowHelpers::handle_fire_timer::h879052abf66da5c6yDu
  19:     0x7f7e005ff8c4 - script_task::ScriptTask::handle_msgs::h5f5776f4259e4bd5TWy
  20:     0x7f7e005f8cc7 - script_task::ScriptTask::start::hfe49fe278eebdf2fJWy
  21:     0x7f7e0013f0f6 - thunk::F.Invoke<A, R>::invoke::h1343387026694508379
  22:     0x7f7e0010e510 - rt::unwind::try::try_fn::h9662973868017941385
  23:     0x7f7e00f17a48 - rust_try_inner
  24:     0x7f7e00f17a35 - rust_try
  25:     0x7f7e0010ec9f - thunk::F.Invoke<A, R>::invoke::h11187635252494347820
  26:     0x7f7e00efdc65 - sys::thread::thread_start::h04d056c6180f4496s8E
  27:     0x7f7dff947529 - start_thread
  28:     0x7f7dfe43722c - clone
  29:                0x0 - <unknown>
```

**Comment by jdm** [comment]
Is this fixed by #5576?

**Comment by metajack** [comment]
Yes, that does indeed fix the problem.
